utilities.js

```diff
@@ -29,15 +29,13 @@ export async function * combineAsyncIterables(iterable1, iterable2) {
 	}
 }
 
-export async function * lineIterator(iterable) {
+export async function * lineIterator(stream) {
+	stream.setEncoding('utf8');
```

Comment:
By default, Node.js streams produce `Buffer` chunks. The chunk might be in the middle of an UTF-8 1-4 bytes sequence. The previous logic would implicitly call `chunk.toString()` (through `buffer += chunk`), leading that partial UTF sequence to be rendered as a question mark string character.

When setting the stream encoding to `utf8`, Node.js handles this automatically. Another way to fix this would be to use `TextDecoder` in stream mode, or `TextDecoderStream`. This results in less code, although it does change the encoding of `subprocess.stdout` and `subprocess.stderr`. But since we're assuming output to be text-based, this is probably a good thing.